pkgs/applications/window-managers/eww/default.nix

```diff
@@ -5,32 +5,26 @@
 , wrapGAppsHook
 , gtk3
 , librsvg
-, withWayland ? false
 , gtk-layer-shell
 , stdenv
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "eww";
-  version = "unstable-2023-08-18";
+  version = "0.5.0";
 
   src = fetchFromGitHub {
     owner = "elkowar";
     repo = "eww";
-    rev = "a9a35c1804d72ef92e04ee71555bd9e5a08fa17e";
-    hash = "sha256-GEysmNDm+olt1WXHzRwb4ZLifkXmeP5+APAN3b81/Og=";
+    rev = "387d344690903949121040f8a892f946e323c472";
+    hash = "sha256-06d8pm1cyg0zpa1kcnzzja2ia3c4glyr6ngkw1hs6l1pivj64kza";
```

Comment:
This should be SRI hash but it looks like "sha256-" + some base32.

Reply:
idk how to generate that hash or cargoHash, i just tried using nix-prefetch-url on the tarball

Reply:
Set the hash to an empty string and run nix build and the hash will be printed.

Reply:
Or `nix-shell '<nixpkgs>' -p nix-prefetch-github --command 'nix-prefetch-github elkowar eww --rev v0.5.0'`